### ISSUE TITLE
auth

### ISSUE THREAD

**Issue by terebentina** [fedecarg/apify-library#3]
The code has limited usability until you add client authentication and user authentication.
Both can be achieved with oauth or with other solutions (headers for api key and signature of request for client auth for example).

Please implement a method of auth.

**Comment by apifytech** [comment]
I agree that client auth is important. It was not included in this first version simply because it was not implemented. If you are building a RESTful Web API then yes, it does have limited usability. The goal of version 1.0 was to focus on the architecture and API. However, if you are building a Web API, then this version should be more than enough.

Future plans include: 
- OAuth support 
- API Management
- API Metrics

**Comment by ifranseda** [comment]
any news on OAuth support?
